cspm/rn/prisma-cloud-compute-release-information/features-introduced-in-compute-january-2023.adoc

```diff
@@ -0,0 +1,682 @@
+[#id-january2023]
+== Features Introduced in January 2023
+
+// Learn about the new Compute capabilities on Prisma™ Cloud Enterprise Edition (SaaS) in January 2023.
+
+The host, container, and serverless capabilities on the *Compute* tab are being upgraded on Jan 22, 2023. When upgraded the version will be 22.12.xxx (TBD).
+
+//TBD: This release includes fixes, and there are no new features in this release.
+
+* xref:#new-features-primsa-cloud-compute[New Features in Prisma Cloud Compute]
+* xref:#idce7ba88f-c752-4509-800d-c29066f34353[API Changes]
+* xref:#id-addressed-issues[Addressed Issues]
+* xref:#id-backward-compatibility[Backward Compatibility for New Features]
+
+[#new-features-primsa-cloud-compute]
+=== New Features in Prisma Cloud Compute
+
+[cols="50%a,50%a"]
+|===
+|Feature
+|Description
+
+2+|*CVE Coverage Update*
+
+|
+| As part of the 22.12 release, Prisma Cloud has rolled out updates to its vulnerability data for Common Vulnerabilities and Exposures (CVEs) in the Intelligence Stream. The new additions are as follows:
+
+* CVEs in Go packages are now detected in the package scope for more accurate results, and not only in the module scope. 
+To read more about Go modules and packages, see https://go.dev/ref/mod#modules-overview[Modules overview].
+
+* Fix date improvement. For any feed collected by IS that does not provide a fix date for CVE, Prisma Cloud Compute will determine the fix date as the date when the fix for the CVE was first seen by the Intelligence Stream.
+
+* Fixed versions enriched for fixed vulnerabilities.
+
+* New PRISMA-IDs have increased 131% since the Kepler major release.
+
+* Fast addition of CVEs (pre-filled CVEs).
++
+CVEs were added to the Intelligence Stream on an average of 13 days before they were analyzed in the NVD.
+As an example, a Kubernetes CVE (CVE-2022-3172) was published on September 16, 2022. The CVE was added to the Prisma Cloud Intelligence stream on September 19, 2022. And at this time in December 2022, the CVE is still reserved in MITRE and not analyzed in NVD. 
+2+|*New Features in the Core Platform*
```

Comment:
@divyabhushan why do we have 2 titles for new features in Core? the second is on line 78

Reply:
@grevach Thanks for pointing that out. I fixed this in the latest commit.